cpp/src/filter/jit/operation-udf.hpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2025, NVIDIA CORPORATION.
```

Reply:
I did resolve it but had to undo it. The CI linter said 2021 should be the date since it is a copy of src/transform/jit/operation-udf.hpp and they share history

Reply:
we'd also need to do the same for rolling and binaryop, probably in a different PR